packages/supabase/test/client_test.dart

```diff
@@ -8,8 +8,9 @@ import 'utils.dart';
 
 void main() {
   group('Standard Header', () {
-    const supabaseUrl = '';
-    const supabaseKey = '';
+    const supabaseUrl = 'https://nlbsnpoablmsiwndbmer.supabase.co';
```

Comment:
I don't think this was intentional.

Reply:
They are actually intentional. I wanted to make sure the URL transformation is properly being done, and an empty string isn't really a realistic test value.